### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-generic.git`](https://github.com/cisagov/skeleton-generic.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/pca-report-library.git pca-report-library
    cd pca-report-library
    git remote add skeleton https://github.com/cisagov/skeleton-generic.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/CODEOWNERS .gitignore .isort.cfg .pre-commit-config.yaml 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Comment by dav3r** [comment]
@BenBreaksThings - I assigned this PR to you, per our recent chat discussion.  Feel free to re-assign to someone else on your team.  Once this PR is done, the next step is to migrate this repo from `skeleton-generic` to `skeleton-python-library`.  Please let me know if you have any questions here.  Thanks!

File: README.md
Comment (dav3r) [inline README.md]: FYI, the block above (`## New Repositories from a Skeleton ## ...`) should be deleted, since it only applies to skeleton repos.

File: README.md
Comment (mcdonnnj) [inline README.md]: The badges in this README do not reference [cisagov/pca-report-library](https://github.com/cisagov/pca-report-library).

File: .github/CODEOWNERS
Comment (mcdonnnj) [inline .github/CODEOWNERS]: This is just a fallback so our team is pulled into reviews that interact with the CI pieces even if we're not `CODEOWNERS` for the repo otherwise.

```suggestion
/.github/ @dav3r @felddy @jsf9k @mcdonnnj
```

File: .github/CODEOWNERS
Comment (mcdonnnj) [inline .github/CODEOWNERS]:
```suggestion
* @ameliav @BenBreaksThings @dav3r @felddy @JCantu248 @jsf9k @mcdonnnj @nickviola
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Outside of the suggestions I made the README needs to have the badge URLs updated to use this project's URL.

File: .github/CODEOWNERS
Comment (dav3r) [reply .github/CODEOWNERS]: This list of usernames should also be alphabetized.

Comment (mcdonnnj) [reply .github/CODEOWNERS]: I've updated my suggestion to be alphabetized so this can be merged from the web interface if desired.

Comment (ameliav) [reply .github/CODEOWNERS]: Thanks all, I will be incorporating these changes but on a brand new branch. Dave suggested that this project use skeleton-docker instead of skeleton-generic so I'm going to pull from there instead, start fresh. I'm going to keep these branches as reference for now.

Comment (dav3r) [reply .github/CODEOWNERS]: > Thanks all, I will be incorporating these changes but on a brand new branch. Dave suggested that this project use skeleton-docker instead of skeleton-generic so I'm going to pull from there instead, start fresh. I'm going to keep these branches as reference for now.

@ameliav - I really think it will be easier for you to continue using this branch/PR to get up-to-date with `skeleton-generic` first, then follow the remaining steps mentioned in [this comment](https://github.com/cisagov/pca-report-library/pull/4#issuecomment-852437603).  If you follow the instructions listed at the top of this PR, you should be able to resolve all of the conflicts and get this PR ready for review.  Please let me know if you have any questions here.

Comment (ameliav) [reply .github/CODEOWNERS]: @dav3r Oh gotcha. I hadn't seen that comment- I'll follow those steps then and let you know if I have questions.

**Comment by ameliav** [comment]
The 1 change requested has been done as well. The README has the badge URLs updated to use this project's URL.

File: README.md
Comment (dav3r) [reply README.md]: Badges corrected/updated in https://github.com/cisagov/pca-report-library/pull/3/commits/0f4cfd76172f40910f30eabe66a4c116104dae69

File: .github/CODEOWNERS
Comment (dav3r) [reply .github/CODEOWNERS]: Corrected as part of https://github.com/cisagov/pca-report-library/pull/3/commits/550dc34b08768dd4ac0d4b38b83a1f6893d4333f

File: README.md
Comment (dav3r) [reply README.md]: The above block was removed as part of https://github.com/cisagov/pca-report-library/pull/3/commits/550dc34b08768dd4ac0d4b38b83a1f6893d4333f

File: src/customer/closing.py
Comment (dav3r) [inline src/customer/closing.py]: This comment can be deleted, since it is covered by the autogenerated `cisagov Libraries` comment above it.
```suggestion
```

**Review by dav3r — COMMENTED** [review]
This looks good to me now, but then again I did a bunch of the commits, so my approval doesn't really count here.

@ameliav - Please ensure that all of the remaining concerns in open conversations have been addressed.  Please reply to the conversation with the commit where the concern was addressed and the original commenter will (hopefully) resolve the conversation.  We don't merge a PR until all open conversations have been resolved.

@jsf9k and @mcdonnnj - please review this PR when you can.

File: src/customer/closing.py
Comment (ameliav) [reply src/customer/closing.py]: Corrected as part of 43ebf41a36b7390834743a1c5d1c711431f8f6f1

File: .github/CODEOWNERS
Comment (ameliav) [reply .github/CODEOWNERS]: Corrected as part of aeac8cefe17c8457b1e7892ed40ffe642549de89

**Review by BenBreaksThings — APPROVED** [review]
Appears to capture all of the things we discussed yesterday getting this up to date, outstanding.

**Review by jsf9k — APPROVED** [review]
This looks correctly merged to me!

File: README.md
Comment (ameliav) [reply README.md]: Hi @mcdonnnj this has been corrected/updated, can you resolve the conversation? (Commit shown in Dave's above comment) Thanks

File: .github/CODEOWNERS
Comment (ameliav) [reply .github/CODEOWNERS]: Hi @mcdonnnj this has been corrected/updated, can you resolve the conversation? (Commit shown in Dave's above comment) Thanks

**Review by mcdonnnj — APPROVED** [review]
I don't see any issues in the core skeleton files. The Python stuff can be left for when you integrate [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library) next. Thanks to your team for working on this! 🌟

**Comment by mcdonnnj** [comment]
@ameliav I have disabled the code owner review requirement so you can merge this PR. I will turn it back on once this PR is merged (and it will be an ongoing requirement).

**Comment by mcdonnnj** [comment]
I'm merging this PR until a permissions discussion is resolved so that others can merge PRs.

**Pull request merged**
